test/declarative/experiments/command-creation-deletion.yaml

```diff
@@ -0,0 +1,200 @@
+## 7. Make sure k8s pod is created for above command
+## 8. Delete Command resource
+## 9. Make sure K8s ConfigMap, Job & Pod related to above
+##    command resource should be deleted
+## 10. Delete clusterrolebinding
+## 11. Delete clusterrole
+## 12. Delete namespace
+apiVersion: dope.mayadata.io/v1
+kind: Recipe
+metadata:
+  name: create-and-assert-command-resource
+  namespace: d-testing
+  labels:
+    d-testing.dope.mayadata.io/inference: "true"
+spec:
+  tasks:
+  - name: create-command-test-namespace
+    create: 
+      state: 
+        kind: Namespace
+        apiVersion: v1
+        metadata:
+          name: recipe-integration-cmd-testing
+  - name: create-service-account
+    create: 
+      state: 
+        kind: ServiceAccount
+        apiVersion: v1
+        metadata:
+          name: recipe-integration-cmd-testing-sa
+          namespace: recipe-integration-cmd-testing
+  - name: create-rbac-clusterrole
+    create: 
+      state: 
+        kind: ClusterRole
+        apiVersion: rbac.authorization.k8s.io/v1
+        metadata:
+          name: recipe-integration-cmd-testing-dope
+        rules:
+        - apiGroups:
+          - "*"
+          resources:
+          - "*"
+          verbs:
+          - "*"
+  - name: create-rbac-cluster-role
+    create: 
+      state: 
+        kind: ClusterRoleBinding
+        apiVersion: rbac.authorization.k8s.io/v1
+        metadata:
+          name: recipe-integration-cmd-testing-dope
+        subjects:
+        - kind: ServiceAccount
+          name: recipe-integration-cmd-testing-sa
+          namespace: recipe-integration-cmd-testing-dope
+        roleRef:
+          kind: ClusterRole
+          name: recipe-integration-cmd-testing-dope
+          apiGroup: rbac.authorization.k8s.io
+  - name: create-command-resource
+    create:
+      state:
+        kind: Command
+        apiVersion: dope.mayadata.io/v1
+        metadata:
+          name: testing-command
+          namespace: recipe-integration-cmd-testing
+        spec:
+          commands:
+          - name: Sleep-test
+            script: sleep 3
+          template:
+            job:
+              apiVersion: batch/v1
+              kind: Job
+              spec:
+                template:
+                  spec:
+                    serviceAccountName: recipe-integration-cmd-testing-sa
+                    containers:
+                    - command:
+                      - /usr/bin/daction
+                      image: mittachaitu/daction:latest
+                      imagePullPolicy: IfNotPresent
+                      name: daction
+                      args:
+                      - -v=3
+                        --command-name=testing-command
+                        --command-ns=recipe-integration-cmd-testing
+  - name: assert-cm-lock-creation
+    assert:
+      state:
+        kind: ConfigMap
+        apiVersion: v1
+        metadata:
+          name: testing-command-lock
```

Comment:
are we going to change the lock name in this PR?
e.g. Will it make use of uid of Command or use generateName property

Reply:
Updated